src/msgflo.cpp

```diff
@@ -404,19 +408,19 @@ shared_ptr<Engine> createEngine(const EngineConfig config) {
             } else {
                 username = up;
             }
-            cout << "username: " << username << endl;
-            cout << "password: " << password << endl;
+            // cout << "username: " << username << endl;
```

Comment:
It should probably be removed, I just left if there mainly because it really isn't battle tested yet.

I can move the logging to the config logging here: https://github.com/trygvis/msgflo-cpp/blob/3fcd3c269cb8e43d445ce9e56a3edbc7f1b3ae55/src/msgflo.cpp#L478